calibration/README.md

```diff
@@ -23,6 +28,8 @@ git clone https://github.com/HabanaAI/vllm-hpu-extension.git
 cd vllm-hpu-extension/calibration
 ```
 3. Download the dataset.
+> [!NOTE]
+> For [DeepSeek-R1](https://huggingface.co/collections/deepseek-ai/deepseek-r1-678e1e131c0169c0bc89728d) series models, you can directly use `NeelNanda/pile-10k` as the dataset.
```

Comment:
maybe it is better to revise from "you can directly use `NeelNanda/pile-10k` as the dataset." to "it is recommended to use `NeelNanda/pile-10k` as the dataset."